### PR TITLE
PSR-402 Added DidTransferIncludeAsset page

### PR DESCRIPTION
**Pull request by westwater**

### CODE REVIEW

**Review by AlessandroCinque — DISMISSED** [review]
LGTM

File: app/controllers/nonsipp/receivetransfer/DidTransferIncludeAssetController.scala [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+  identifyAndRequireData: IdentifyAndRequireData,
+  formProvider: YesNoPageFormProvider,
+  val controllerComponents: MessagesControllerComponents,
+  view: YesNoPageView
+)(implicit ec: ExecutionContext)
+    extends PSRController {
+
+  private val form = DidTransferIncludeAssetController.form(formProvider)
+
+  def onPageLoad(srn: Srn, index: Max300, secondaryIndex: Max50, mode: Mode): Action[AnyContent] =
```
Comment (kberec) [inline app/controllers/nonsipp/receivetransfer/DidTransferIncludeAssetController.scala:55]: Should this be Max5?

File: app/pages/nonsipp/receiveTransfer/DidTransferIncludeAssetPage.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import pages.QuestionPage
+import config.Refined._
+import utils.RefinedUtils._
+import eu.timepit.refined.refineMV
+
+case class DidTransferIncludeAssetPage(srn: Srn, index: Max300, secondaryIndex: Max50) extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ toString \ index.arrayIndex.toString \ secondaryIndex.arrayIndex.toString
+
+  override def toString: String = "didTransferIncludeAsset"
```
Comment (kberec) [inline app/pages/nonsipp/receiveTransfer/DidTransferIncludeAssetPage.scala:31]: transferIncludedAsset ?

File: app/controllers/nonsipp/receivetransfer/DidTransferIncludeAssetController.scala (hunk 1 above)
Comment (westwater) [reply app/controllers/nonsipp/receivetransfer/DidTransferIncludeAssetController.scala:55]: Narendra is doing that as part of his ticket so I don't have access to the index yet

**Review by kberec — APPROVED** [review]

**Pull request merged**